### PR TITLE
SSB: Add KingSwordYT

### PR DESCRIPTION
**Pull request by CheeseMuffin**

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -306,6 +306,33 @@ let BattleMovedex = {
...
+		name: "Dragon Warrior Touch",
+		pp: 5,
+		priority: 0,
+		flags: {protect: 1, mirror: 1, punch: 1, contact: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, 'Outrage', target);
+			this.add('-anim', source, 'Drain Punch', target);
+		},
+		secondary: {
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:270]: Don't mark it as a secondary if its 100%, `self:{ boosts: {atk: 1}}` will work

File: mods/seasonal/abilities.js [hunk 2]
```diff
@@ -42,6 +42,23 @@ let BattleAbilities = {
...
 		},
 	},
+  // KingSwordYT
+	kungfupanda: {
+		desc: "This Pokemon's punch-based attacks have their power multiplied by 1.2, and this Pokemon's Speed is raised by 1 stage after it is damaged by a move",
+		shortDesc: "This Pokemon's punch-based attacks have 1.2x power. +1 Spe when hit.",
+		onBasePowerPriority: 8,
+		onBasePower: function (basePower, attacker, defender, move) {
+			if (move.flags['punch']) {
+				this.debug('Iron Fist boost');
```
Comment (HoeenCoder) [inline mods/seasonal/abilities.js:52]: Should be `Kung Fu Panda boost`

**Comment by HoeenCoder** [comment]
Not sure what you did here but the history seems messed up

**Comment by CheeseMuffin** [comment]
Deleting this pr since I messed up the history

**Pull request closed without merging**